boards/arm/nrf52810_pca10040/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Kconfig - nRF52810 PCA10040 board configuration
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF52810_PCA10040
```

Comment:
here too, and everywhere else

Reply:
same

Reply:
By the way @carlescufi do we really need these empty Kconfig files?